tests/test_routes.py

```diff
@@ -85,3 +89,28 @@ def test_create_order_error(self):
             content_type="application/x-www-form-urlencoded")
         self.assertEqual(resp.status_code, status.HTTP_415_UNSUPPORTED_MEDIA_TYPE)
 
+    @patch('flask_sqlalchemy._QueryProperty.__get__')
+    def test_add_item(self, sql_alchemy_get_mock):
+        """Test adding items to the order."""
+        # Create an order.
+        db.session.commit = MagicMock(return_value=None)
```

Comment:
Same question here. Doesn't this prevent us from actually committing to the database?

Reply:
Mock doesn't contradicts and aligned with professor lectures. Let us do that from next time after discussing this with Prof. I am okay with other tests being added that follow sample code so in the interest of time let us keep them in this pull request.